Sprint-1/destructuring/exercise-3/exercise.js

```diff
@@ -6,3 +6,18 @@ let order = [
   { itemName: "Hot Coffee", quantity: 2, unitPricePence: 100 },
   { itemName: "Hash Brown", quantity: 4, unitPricePence: 40 },
 ];
+
```

Comment:
Good work, please read what you have been asked, In this example they showed you what the expected result look like. 
You have to show all of the things you have been asked to do

Reply:
thank you, added padEnd() so now the result looks as expected